model/item.cc

```diff
@@ -31,22 +31,21 @@
 #include <exception>
 
 using ecolab::Pango;
-using namespace std;
```

Comment:
Any particular reason for not importing std namespace here?

Reply:
Oops, I was working on both operation.cc and item.cc at the same time and added it to operation.cc but then deleted it from item.cc by accident. I will put it back in.

Reply:
I have tried several different approaches, but I cannot reconcile BottomRightResizerItem::onResizeHandles() with BottomRightResizerItem::drawResizeHandles() for a general rotation and I don't understand why. If I rotate cairo in BoundingBox::update before drawing, then the resize arrows appear where they are supposed to but cannot be selected for variables that have names longer than one letter. If I rotate after drawing in BoundingBox::update, I cannot get the resize arrows to appear where they are supposed to. So, I'm closing this PR which might have to go on the backlog if I can't fix it before this iteration finalises. :(